Godeps/Godeps.json

```diff
@@ -435,82 +462,82 @@
 		},
 		{
 			"ImportPath": "github.com/vmware/govmomi",
-			"Comment": "v0.1.0-454-g9051bd6",
+			"Comment": "v0.6.2",
```

Comment:
Not entirely sure. I just ran the steps in CONTRIBUTING.md to update a library:

```
make dep-restore
go get -u github.com/aws/aws-sdk-go
make dep-save
```

I assumed any version changes were transitive dependencies.